### ISSUE TITLE
Disable password reset on a user

### ISSUE THREAD

**Issue by jaywood128** [FusionAuth/fusionauth-issues#2072]
### Problem
Using the FusionAuth-hosted login page, I want to only allow a user to login with an Idp or a SAML provider. I could disable credential login by removing the login form, and setting the password to a random value the user is not aware of, but this would not prevent them from attempting to change their password.

This would be helpful if you had customers who want to effectively force SSO while preventing users from changing their passwords. 

### Solution
Provide an option to disable a user from changing their password. 

### Alternatives/workarounds
As stated above, remove the login form and set the user's password to a random value.

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by robotdan** [comment]
Assuming this is a customer request - would this be an application scoped configuration, or would it be per user, or a policy to define a group of users?

**Comment by epbensimpson** [comment]
For our use case, either per-user or per-group would work. We have a single application, used by both our staff and our customers. Our staff should be required to sign in with Google, and our customers should be able to configure this for their organisation's users. So an application-scoped option would not be suitable for us.

**Comment by krajncmitja** [comment]
I would agree with option to have this per user. Although we have several applications, they are used primarily for whitelabeling purpose. A simple flag on user which would allow or deny starting the forgot password process would be great for our case.

**Comment by mattwyatt-recurve** [comment]
We are also looking for this option. Have any workarounds been suggested since this issue was raised? Is this feature on any roadmap?

**Comment by mooreds** [comment]
Might be related to https://github.com/fusionauth/fusionauth-issues/issues/1

**Comment by mooreds** [comment]
In 1.59.0, we delivered the ability to have a user without a password, resolving https://github.com/FusionAuth/fusionauth-issues/issues/484 

Folks who chimed in above, does this address your needs?
